my-school/src/Components/ActivityCard.js

```diff
@@ -242,10 +242,10 @@ function ActivityCard(props)  {
                                         </NumberInput>
                                     </FormControl>
                                 </Flex>
-                                <h3 style={{ fontWeight: "bold", width: '100%', paddingTop: '1rem', paddingBottom: '.5rem' }}>Date Completed:</h3>
+                                {/* <h3 style={{ fontWeight: "bold", width: '100%', paddingTop: '1rem', paddingBottom: '.5rem' }}>Date Completed:</h3>
                                 <Flex flexWrap='wrap' borderWidth="1px" rounded="lg" w='300px' padding='1rem' w='400px' justifyContent='center' >
                                     <div><DateSelector w="100%" /></div>
-                                </Flex>
+                                </Flex> */}
```

Comment:
If we don't want this, feel free to delete. It lives on in github and will make master more readable to just pull it out.

Reply:
Someone's going to EVENTUALLY delete something they shouldn't. We might as well normalize it and get used to recovering via git.